server/src/test/java/org/elasticsearch/action/bulk/TransportBulkActionIngestTests.java

```diff
@@ -404,7 +411,7 @@ public void testIngestForward() throws Exception {
         ActionTestUtils.execute(action, null, bulkRequest, listener);
 
         // should not have executed ingest locally
-        verify(ingestService, never()).executeBulkRequest(anyInt(), any(), any(), any(), any(), any());
+        verify(ingestService, never()).processBulkRequest(any(), anyInt(), any(), any(), any(), any(), any());
```

Comment:
I decided to rename the method even if that changed a lot of test code - IMO this better describes what the method does as it does not actually execute the bulk request, but preprocess it before executing